content/pages/about.html

```diff
@@ -0,0 +1,396 @@
+        Control who has access to your documentation with integrated authentation.
+      {% endcall %}
+
+      {% call homepage.feature_detail_item(header="Bring your domain", icon="fad fa-at") %}
+        Use your own custom domain for your hosted documentation, with SSL automatically configured.
+      {% endcall %}
+
+      {% call homepage.feature_detail_item(header="Integrated search", icon="fad fa-search") %}
+        Help your users find what the need with integrated search across all your projects and versions.
+      {% endcall %}
+
+      {% call homepage.feature_detail_item(header="Redirects", icon="fad fa-signs-post") %}
+        Avoid broken links by adding redirects for pages that have been moved or deleted. 
+      {% endcall %}
+
+    </div>
+  </div>
+</section>
+
+<section>
+  <div class="ui padded container">
+    <div class="ui vertically padded centered grid">
+      <div class="center aligned column">
+        <div class="ui very padded raised inverted secondary large segment">
+          <p>
+            Start automating your documentation with Read the Docs.
+          </p>
+          <p>
+            <button
+               class="ui large inverted button"
+               onclick="jQuery('#about-modal').modal('show');"
+               type="button">
+              <i class="fad fa-play icon"></i>
+              Get started now
+            </button>
+          </p>
+
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+{% block homepage_modal %}
+  {{ homepage.signup_modal("about-modal") }}
+{% endblock homepage_modal %}
+
+
+{{ about_divide("Additional features") }}
+
+<section>
+  <div class="ui very padded container" id="more">
+    <div class="ui vertically padded centered grid">
+
+      {% call homepage.feature_detail_item(header="Automation rules", icon="fad fa-robot") %}
+        Automatically increment your documentation versions, and rebuild on new version releases.
+      {% endcall %}
+
+      {% call homepage.feature_detail_item(header="Subprojects", icon="fad fa-folder-tree") %}
+        Subprojects can share a search index and a custom domain, and can help logically group sets of multiple projects.
+      {% endcall %}
+
+      {% call homepage.feature_detail_item(header="Environment variables", icon="fad fa-shield-keyhole") %}
+        Use environment variables to provide secrets to your project's build commands.
+      {% endcall %}
+
+      {% call homepage.feature_detail_item(header="Analytics", icon="fad fa-chart-column") %}
+        Track search query and documentation page traffic data with no additional services.
+      {% endcall %}
+
+      {% call homepage.feature_detail_item(header="Webhooks", icon="fad fa-satellite-dish") %}
+        Webhooks can be sent after your project builds, so you can add new functionality.
+      {% endcall %}
+
+      {% call homepage.feature_detail_item(header="Notifications", icon="fad fa-satellite-dish") %}
```

Comment:
I would use https://fontawesome.com/icons/bullhorn?s=solid or https://fontawesome.com/icons/bell?s=regular for this icon 😄

Reply:
Good point, maybe https://fontawesome.com/icons/bell-on?s=duotone or https://fontawesome.com/icons/bell?s=duotone or is the most recognizable

Reply:
"bell-on" works pretty good!

Reply:
Actually, a much better fit is https://fontawesome.com/icons/envelope-dot

I also updated the copy there, I think I meant to replace the copy on both sections but lost track.